services/ui-src/src/components/statusing/TableRow.tsx

```diff
@@ -0,0 +1,83 @@
+import { Td, Text, Tr } from "@chakra-ui/react";
+import { ReportPageProgress, ReportType } from "types";
+import { useBreakpoint, useStore } from "utils";
+import { EditButton } from "./EditButton";
+import { StatusIcon } from "./StatusIcon";
+
+export const TableRow = ({ page, rowDepth }: RowProps) => {
+  const { isMobile } = useBreakpoint();
+  const { name, path, children, status } = page;
+  const { report, editable } = useStore();
+  const buttonAriaLabel = editable ? `Edit  ${name}` : `View  ${name}`;
+
+  const isMobileAndNotChildEditButton =
+    isMobile && !children && EditButton({ buttonAriaLabel, path, editable });
+
+  const notChildEditButton =
+    !children &&
+    EditButton({ buttonAriaLabel, path, editable, showIcon: true });
+
+  const parentPl = !isMobile ? "1rem" : "0";
+  const subparentPl = !isMobile ? `${1.25 * rowDepth}rem` : "0";
+
+  const ptRowDepth1 = isMobile ? "1.5rem" : "0.5rem";
+  const ptRowDepthOver1 = isMobile ? "1rem" : "0.5rem";
+
```

Comment:
I tried to make the `return` code block easier to read but I feel like there's a nicer way to do this. Open to suggestions

Reply:
Instead of ternaries, I prefer one conditional block. I also try to avoid negatives for readability, if everything can go under `isMobile`.
```
let parentPl = "1rem";
let subparentPl = `${1.25 * rowDepth}rem`;
etc.

if (isMobile) {
   parentPl = "0";
   subparentPl = "0";
   etc.
}

```